core/src/main/java/com/radixdlt/RadixNodeApplication.java

```diff
@@ -86,32 +84,13 @@ private RadixNodeApplication() {}
     System.setProperty("java.net.preferIPv4Stack", "true");
   }
 
-  private static final Object BC_LOCK = new Object();
-  private static boolean bcInitialised;
-
-  private static void setupBouncyCastle() {
-    synchronized (BC_LOCK) {
-      if (bcInitialised) {
-        log.warn("Bouncy castle is already initialised");
-        return;
-      }
-
-      Security.insertProviderAt(new BouncyCastleProvider(), 1);
-      bcInitialised = true;
-    }
-  }
-
   public static void main(String[] args) {
     try {
       MemoryLeakDetector.start();
-
       logVersion();
       dumpExecutionLocation();
-      // Bouncy Castle is required for loading the node key, so set it up now.
-      setupBouncyCastle();
       final var properties = RuntimeProperties.fromCommandLineArgs(args);
       bootstrapRadixNode(properties);
-      log.info("Node bootstrapping completed");
```

Comment:
Out of interest, why was this removed?

Reply:
we already have a similar message in RadixNodeApplication:
```Radix node {} started successfully in {} ms```